src/openforms/upgrades/upgrade_paths.py

```diff
@@ -76,6 +76,12 @@ def run_checks(self) -> bool:
 # If your current version falls outside of a supported range, you need to do another
 # upgrade path (first) or there is no upgrade path at all.
 UPGRADE_PATHS = {
+    "2.7": UpgradeConstraint(
+        valid_ranges={
+            VersionRange(minimum="2.6"),
```

Comment:
Not sure about the minimum version range

Reply:
I have the right one in https://github.com/open-formulieren/open-forms/pull/4446 :)

Reply:
I've merged the squashed migrations PR, so you can rebase on master to add the check!

Reply:
thanks